less/components/sidebar.less

```diff
@@ -14,36 +14,25 @@ html.no-js .sidebar {
 }
 
 .page {
-  &.events {
-    .sidebarColorizer(@darkBlueSidebarColor);
-  }
+  &.events,
   &.event-resources {
     .sidebarColorizer(@darkBlueSidebarColor);
   }
-  &.teach-like-mozilla {
-    .sidebarColorizer(@greenSidebarColor);
-  }
-  &.tools-page {
+  &.teach-like-mozilla,
+  &.tools-page,
+  &.web-literacy {
     .sidebarColorizer(@greenSidebarColor);
   }
-  &.clubs {
-    .sidebarColorizer(@orangeSidebarColor);
-  }
+  &.clubs,
   &.clubs-toolkit {
     .sidebarColorizer(@orangeSidebarColor);
   }
-  &.teaching-materials {
+  &.teaching-materials,
+  &.web-lit-basics,
+  &.protect-your-data,
+  &.maker-party-2015 {
     .sidebarColorizer(@pinkSidebarColor);
   }
-  &.web-lit-basics {
-    .sidebarColorizer(@pinkSidebarColor);
-  }
-  &.protect-your-data {
-    .sidebarColorizer(@pinkSidebarColor);
-  }
-  &.web-literacy {
-    .sidebarColorizer(@greenSidebarColor);
-  }
 
```

Comment:
I added `&.maker-party-2015` and grouped classes that share the same sidebar colour together